tests/scripts/all.sh

```diff
@@ -3169,6 +3169,18 @@ component_test_psa_crypto_config_chachapoly_disabled() {
     make test
 }
 
+component_test_ccm_aes_sha256() {
+    msg "build: CCM + AES + SHA256 configuration"
+
+    cp tests/include/test/drivers/config_test_driver.h  include/mbedtls/mbedtls_config.h
+    cp configs/crypto-config-ccm-aes-sha256.h           include/psa/crypto_config.h
```

Comment:
We should use `"$CONFIG_H"`, `"$CRYPTO_CONFIG_H"` and `"$CONFIG_TEST_DRIVER_H"` here